src/theme/base.scss

```diff
@@ -7,8 +7,8 @@ body,
 html {
   font-family: var(--theme-font);
   font-size: 100%;
-  background-color: var(--theme-color-paper);
-  color: var(--theme-color-ink);
+  background-color: var(--theme-color-blue-light);
+  color: var(--theme-color-blue-navy);
```

Comment:
@youweb3  these colours no long exist since you have changed the naming of your colour variables. Please update these before merging.